x-pack/plugins/screenshotting/server/browsers/chromium/driver_factory/index.ts

```diff
@@ -114,44 +127,64 @@ export class HeadlessChromiumDriverFactory {
     const dataDir = getDataPath();
     fs.mkdirSync(dataDir, { recursive: true });
     this.userDataDir = fs.mkdtempSync(path.join(dataDir, 'chromium-'));
+    logger.debug(`Chromium userDataDir: ${this.userDataDir}`);
```

Comment:
This is not passed to `logs$.next` because those logs are spit out in the diagnostic API response, and this is a bit sensitive since it contains an absolute path.